idl/parser_test.go

```diff
@@ -123,7 +131,7 @@ func TestParseErrors(t *testing.T) {
 				enum Foo {}
 				include "bar.thrift"
 			`,
-			wantErrors: []string{"line 3:5", "unexpected INCLUDE"},
+			wantErrors: []string{"line 3:33", "unexpected INCLUDE"},
```

Comment:
These cases aren't are obvious because of the way Go's multiline string literals work in combination with the indentation we're using here.

I think this is fine, but we could either skip the indentation or use something like https://github.com/MakeNowJust/heredoc/.